package.json

```diff
@@ -625,6 +655,30 @@
               "trace"
             ],
             "default": "info"
+          },
+          "debuggerLogLevel": {
```

Comment:
So it gets defined twice because [contributes.configuration](https://code.visualstudio.com/api/references/contribution-points#contributes.configuration) are `Contribute configuration keys that will be exposed to the user.`, see link. Then [contributes.debuggers.configurationAttributes](https://code.visualstudio.com/api/references/contribution-points#contributes.debuggers) is the schema for launch configuration arguments specific to this debugger.`, see link.